### DISCUSSION TITLE
diary/2023/12/31/2023-retrospective/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#152] [Comments]
# diary/2023/12/31/2023-retrospective/

2023년도 이제 마무리할 시기가 다가옵니다. 여러분들의 2023년은 어떠셨나요? 매년 이 시기엔 1년은 마무리하며 회고하고 있습니다. 회고의 목적은 제 1년을 간단히 글로 정리하기 위한 목적과 누군가 저의 2023년(저의 7년차)에 대해 물어보면 공유하기 위한 목적입니다 이 글을 보시는 분들도 여러분들의 1년을 생각하시면서 읽으시면 좋을 것 같아요 저는 회고 글을 작성하기 전에 과거에...

https://zzsza.github.io/diary/2023/12/31/2023-retrospective/

**Comment by emjemoon** [comment]
인사이트가 넘 많네요! 잘 읽었습니다, 공유해주셔서 감사합니다. 새해 복 많이 받으세요 :)

**Reply by zzsza** [reply]
인사이트가 많다고 해주셔서 감사합니다! 도움이 되면 좋겠어요.
새해 복 많이 받으셔요 :)

**Comment by standing-o** [comment]
항상 잘보고 있습니다! 새해 복 많이 받으세용

**Reply by zzsza** [reply]
항상 글 봐주셔서 감사합니다! 새해 복 많이 받으셔요 :)

**Comment by ChoiInYeol** [comment]
좋은 글 정말 감사합니다 :)

**Reply by zzsza** [reply]
좋은 글이라 해주셔서 감사합니다! 새해 복 많이 받으셔요!
